### ISSUE TITLE
Single workflow event lifetime

### ISSUE THREAD

**Issue by Eitanski** [elsa-workflows/elsa-core#5203]
I want that once an event is raised, and has started unblocking, the same event will not be raised, which means that the next workflow instance will block on this same event activity.

Currently this is not the case, a single event remains raised for several workflow executions.

Also, when I publish the event I do not want to resolve the current workflow instance, I saw that it is an option.

**Comment by sfmskywalker** [comment]
This is fixed as part of PR https://github.com/elsa-workflows/elsa-core/pull/5444 - We can close this issue when that PR gets merged into `main`.
